discord/ext/test/factories.py

```diff
@@ -245,13 +237,18 @@ def _mention_from_channel(channel):
 
     return out
 
+def _mention_from_role(role):
```

Comment:
This seems wrong, as the implementation of `_handle_mention_roles` in message.py does `map(int, role_mentions)`. Why the dict?

Reply:
yup, I was matching the others and didn't bother to notice this one is just ints.  Will fix.